examples/benchmarks/Localformer/workflow_config_localformer_Alpha360.yaml

```diff
@@ -0,0 +1,82 @@
+qlib_init:
+    provider_uri: "~/.qlib/qlib_data/cn_data"
+    region: cn
+market: &market csi300
+benchmark: &benchmark SH000300
+data_handler_config: &data_handler_config
+    start_time: 2008-01-01
+    end_time: 2020-08-01
+    fit_start_time: 2008-01-01
+    fit_end_time: 2014-12-31
+    instruments: *market
+    infer_processors:
+        - class: FilterCol
+          kwargs:
+              fields_group: feature
+              col_list: ["RESI5", "WVMA5", "RSQR5", "KLEN", "RSQR10", "CORR5", "CORD5", "CORR10", 
```

Comment:
this  configuration doesn't work due to the features don't  align with Alpha360

Reply:
same problem with  examples/benchmarks/Transformer/workflow_config_transformer_Alpha360.yaml

Reply:
Sure. I will fix it.

Reply:
I have updated the files. Now, it can work.